toolkit/tools/imagecustomizer/docs/configuration.md

```diff
@@ -1171,6 +1171,20 @@ For example, `/boot` will be mounted before `/boot/efi`.
 
 Points to a script file (typically a Bash script) to be run during customization.
 
+Scripts are run with a limited set of capabilities. Specifically:
```

Comment:
nit - maybe add a hyperlink to https://man7.org/linux/man-pages/man7/capabilities.7.html ?

Reply:
Fixed.